windows/src/developer/TIKE/project/Keyman.Developer.UI.Project.UfrmNewModelProjectParameters.pas

```diff
@@ -179,6 +179,12 @@ procedure TfrmNewModelProjectParameters.FormCreate(Sender: TObject);
   inherited;
   editPath.Text := FKeymanDeveloperOptions.DefaultProjectPath;
 
+  dlgBrowse := TBrowse4Folder.Create(Self);
+  dlgBrowse.InitialDir := editPath.Text;
+  dlgBrowse.Options := [OnlySelectFileSysAncestors, ShowEditBox, UseNewDialogStyle];
+  dlgBrowse.Root := Desktop;
```

Comment:
Was there a workflow of having Developer consolidate projects to
`Documents\Keyman Developer\Projects\`?

Reply:
Yes, see above -- `editPath.Text := FKeymanDeveloperOptions.DefaultProjectPath`. The Root is simply the root folder for the browser folder dialog -- not the selected folder.